.github/workflows/easy-cp.yml

```diff
@@ -45,31 +45,29 @@ jobs:
         run: |
           git config user.name "GitHub Actions Bot"
           git config user.email "<>"
-          git fetch origin $RELEASE_BRANCH
-          git fetch origin master
-          git checkout -b $RELEASE_BRANCH --track origin/$RELEASE_BRANCH
+          git remote add upstream https://github.com/flutter/flutter.git
+          git fetch upstream $RELEASE_BRANCH
+          git fetch upstream master
+          git checkout -b cp-${CHANNEL}-${COMMIT_SHA} --track upstream/$RELEASE_BRANCH
           git cherry-pick $COMMIT_SHA
       # TODO(xilaizhang): remove this step once the template is available on release branches.
       - name: Get CP Template
         run: |
           curl -o PULL_REQUEST_CP_TEMPLATE.md https://raw.githubusercontent.com/flutter/flutter/master/.github/PR_TEMPLATE/PULL_REQUEST_CP_TEMPLATE.md
       - name: Create PR on CP success
         if: ${{ steps.attempt-cp.conclusion == 'success' }}
-        uses: peter-evans/create-pull-request@153407881ec5c347639a548ade7d8ad1d6740e38
-        with:
-          base: ${{ env.RELEASE_BRANCH }}
-          branch: cp-${{ env.CHANNEL }}-${{ env.COMMIT_SHA }}
-          path: flutter
-          token: ${{ secrets.FLUTTERACTIONSBOT_CP_TOKEN }}
-          labels: |
-            cp: review
-          title: '[${{ env.CHANNEL }}-cherrypick] cherrypicks commit ${{ env.COMMIT_SHA }} from PR ${{ github.event.pull_request.title }}'
-          body-path: PULL_REQUEST_CP_TEMPLATE.md
+        working-directory: ./flutter
+        run: |
+          git push -u origin cp-${CHANNEL}-${COMMIT_SHA}
+          gh pr create --title "[cp-${CHANNEL}]${PR_TITLE}" --body-file ../PULL_REQUEST_CP_TEMPLATE.md --base ${RELEASE_BRANCH} --label "cp: review" --repo flutter/flutter --head flutteractionsbot:cp-${CHANNEL}-${COMMIT_SHA}
```

Comment:
At the start of the title, `[cp-` -> `[CP-`

Acronyms are capitalized to distinguish them from normal words.

Reply:
TIL, thanks for pointing this out!